dataset/pipeline.py

```diff
@@ -187,7 +187,7 @@ def __getattr__(self, name):
         if name[:2] == '__' and name[-2:] == '__':
             # if a magic method is not defined, throw an error
             raise AttributeError()
-        elif self._is_batch_method(name):
+        elif self._is_batch_method(name, cls=self.dataset.batch_class):
```

Comment:
That would limit the search to children of `dataset.batch_class` which may not be desirable in many cases (e.g. an action of `radio.CTImagesBatch` can create an instance of `ImagesBatch` but the latter is not a child of the former.

Reply:
I've got use-case where it crashed. Let me try to repeat